src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.data.task.TaskList;
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.message.Messages;
+
+public class ListCommand extends Command {
+    /**
+     * Show a list of all tasks in the tasklist
+     *
+     * @param tasks   The list of tasks known
+     * @param storage The storage that is used
+     * @param ui The ui that is used
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        if (tasks.size() == 0) {
+            System.out.println("No task added yet!");
+        } else {
+            System.out.println(String.format(Messages.MESSAGE_LIST, tasks));
```

Comment:
Avoid doing complex operations in print statements. The sole purpose of print statements is to print and not to carry out operations along with printing. Please separate this logic into different lines.